### ISSUE TITLE
finish wrapper for sbpl

### ISSUE THREAD

**Issue by jcmayoral** [jcmayoral/gr_navigation#52]
Use SBPL to generate time path planning... using it as a different node will allow to split the move_base navigation stack doing the first step for a new navigation pipeline..

Enhancements will include:

-> Get rid of the local costmap, global might be still needed but it will depend on issue #51 ... Safety systems will be running on a parallel threat.
-> Path planning based on kinematics of a robot.
-> After some future work... we might reach performance analysis on real time.
-> If primitive script is parsed to python.. we might be able to train the path planning generation according to thorvald configuration (open to discussion).

DIsadvantage:

-> Buggy source code (I tried before to dig in it during my internship).
-> Hard to implement.
-> We might be the first on implementing this on outdoors.

Good News:
-> I have some scripts to create near optimal parameters for primitives generation.
